report-pox-krypton.js

```diff
@@ -771,16 +779,59 @@ function process_burnchain_ops() {
     console.log(`${miner_key}/${c32.c32ToB58(miner_key)} ${miner.actual_win}/${miner.won}/${miner.mined} ${(miner.won / miner.mined * 100).toFixed(2)}% ${(miner.actual_win / actual_win_total * 100).toFixed(2)}% - ${miner.burned} - Th[${(miner.burned / miner.total_burn * 100).toFixed(2)}%] (${miner.burned / miner.mined})`)
     miner.average_burn = miner.burned / miner.mined
     miner.normalized_wins = miner.won / miner.average_burn
+    miner.burn_per_win = miner.burned / miner.won;
+    miner.profit = miner.won * 500 * 0.15 - (miner.burned / 100000000) * 13000;
   }
 
-  console.log("========================================================================================================================")
-  for (let miner_key of Object.keys(miners).sort((a, b) => (miners[b].normalized_wins - miners[a].normalized_wins))) {
-    const miner = miners[miner_key]
-    console.log(`${miner_key}/${c32.c32ToB58(miner_key)} ${miner.actual_win}/${miner.won}/${miner.mined} ${(miner.won / miner.mined * 100).toFixed(2)}% ${(miner.actual_win / actual_win_total * 100).toFixed(2)}% - ${miner.burned} - Th[${(miner.burned / miner.total_burn * 100).toFixed(2)}%] (${miner.burned / miner.mined}) ${miner.normalized_wins}`)
+  console.log(
+    "========================================================================================================================"
+  );
+  for (let miner_key of Object.keys(miners).sort(
+    (a, b) => miners[b].normalized_wins - miners[a].normalized_wins
+  )) {
+    const miner = miners[miner_key];
+    console.log(
+      `${miner_key}/${c32.c32ToB58(miner_key)} ${miner.actual_win}/${
+        miner.won
+      }/${miner.mined} ${((miner.won / miner.mined) * 100).toFixed(2)}% ${(
+        (miner.actual_win / actual_win_total) *
+        100
+      ).toFixed(2)}% - ${miner.burned} - Th[${(
+        (miner.burned / miner.total_burn) *
+        100
+      ).toFixed(2)}%] (${miner.burned / miner.mined}) ${
+        miner.normalized_wins
+      } ${miner.profit}`
+    );
   }
 
-
-})()
-
+  console.log(
+    "========================================================================================================================"
+  );
+
+  const burnPerWinFormat = new Intl.NumberFormat("en-US", {
+    style: "decimal",
+    maximumFractionDigits: 2,
+    minimumFractionDigits: 2,
+  });
+  const profitFormat = new Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+  });
+  for (let miner_key of Object.keys(miners).sort(
+    (a, b) => miners[a].burn_per_win - miners[b].burn_per_win
+  )) {
+    const miner = miners[miner_key];
+    console.log(
+      `${miner_key.padEnd(41, " ")}/${c32.c32ToB58(
+        miner_key
+      )} ${burnPerWinFormat
+        .format(miner.burn_per_win)
+        .padStart(14, " ")} ${profitFormat
+        .format(miner.profit)
+        .padStart(14, " ")}`
+    );
+  }
+})();
```

Comment:
interesting stat.  Will need to add more options so we can get more granular about what list you want to see.  But that can be done afterward